src/winetricks

```diff
@@ -11640,11 +11640,12 @@ load_speechsdk()
 
     # If sapi.dll isn't in original location, applications won't start, see
     # e.g., https://bugs.winehq.org/show_bug.cgi?id=43841
-    w_try ln -s "$W_COMMONFILES_X86/Microsoft Shared/Speech/sapi.dll" "$W_32BIT_DLLS/Speech/Common"
+    mkdir -p "$W_SYSTEM32_DLLS/Speech/Common/"
+    w_try ln -s "$W_COMMONFILES_X86/Microsoft Shared/Speech/sapi.dll" "$W_SYSTEM32_DLLS/Speech/Common"
 
     w_override_dlls native sapi
 
-    w_set_winver 'default'
+    w_set_winver winxp
```

Comment:
Please add a comment here saying why we need winxp. Otherwise LGTM, thanks!

Reply:
@austin987 done!